guides/intergration/oidc.md

```diff
@@ -0,0 +1,120 @@
+# OpenID Connect
+
+Trento integrates with identity providers that use the OpenID Connect **(OIDC)** protocol to authenticate users accessing the console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external identity provider **(IDP)**.
```

Comment:
Strange that you put the acronymi in the second appereance of identity provider rather than the first one almost at the beginning